LayoutTests/imported/w3c/web-platform-tests/storage/storagemanager-persist-persisted-match.https.any.js

```diff
@@ -0,0 +1,13 @@
+// META: title=StorageManager: results of persist() and persisted() are the same
+
+promise_test(function() {
+    var persistResult = false;
+    return navigator.storage.persist().then(function(result) {
```

Comment:
async/await would be more natural.
Since it is a WPT test, we should upstream these changes before merging this PR or move it to LayoutTests/http/wpt/.